lib/cypress/name_randomizer.rb

```diff
@@ -31,5 +35,13 @@ def self.replace_random_char(name, random: Random.new)
       name[name_pos] = name[name_pos] != lsamples[0] ? lsamples[0] : lsamples[1]
       name
     end
+
+    def self.safe_nickname(nicknames, given_name, family_name, patients, random: Random.new)
+      # if no nicknames, use first initial
```

Comment:
Are we worried about two names with no nicknames that result in the same first initial used?